websites/K/Kinopoisk/presence.ts

```diff
@@ -187,42 +181,66 @@ presence.on("UpdateData", async () => {
 				)?.toLowerCase()}`;
-				presenceData.state = document.querySelector<HTMLImageElement>(
-					".FilmContent_wrapper__EicQU img"
-				)?.alt;
+				presenceData.state = document
+					.querySelector<HTMLImageElement>(".FilmContent_wrapper__EicQU img")
+					?.alt.replace("Смотреть", "")
+					.replace("фильм", "");
+				presenceData.smallImageKey = Assets.Viewing;
 			}
 
 			if (document.querySelector(".CrispySlideDown_fade_active__StELV")) {
 				presenceData.details = `Смотрит информацию ${textContent(
 					".CrispySlideDown_fade_active__StELV .TabList_root__Kwcez button"
 				)?.toLowerCase()}`;
-				presenceData.state = document.querySelector<HTMLImageElement>(
-					".CrispySlideDown_fade_active__StELV img"
-				)?.alt;
+				presenceData.state = document
+					.querySelector<HTMLImageElement>(
+						".CrispySlideDown_fade_active__StELV img"
+					)
+					?.alt.replace("Смотреть", "")
+					.replace("фильм", "");
+				presenceData.smallImageKey = Assets.Viewing;
 			}
 
 			if (document.querySelector(".PlayerManager_body__rOEVd")) {
-				presenceData.details = `Смотрит ${
-					!privacy
-						? pageTitle(" — ")[0]
-						: textContent(".Meta_subtitle__jnooi")
-						? "сериал"
-						: "фильм"
-				}`;
-				presenceData.state = textContent(".Meta_subtitle__jnooi");
-				presenceData.smallImageKey = video.paused ? Assets.Pause : Assets.Play;
-				presenceData.smallImageText = video.paused
-					? strings.pause
-					: strings.play;
-
-				if (video.paused || !time) {
+				if (document.querySelector(".Meta_image__CXoKi")) {
+					contentTitle =
+						document.querySelector<HTMLImageElement>(".Meta_image__CXoKi")?.alt;
+				}
+
+				if (document.querySelector(".Meta_subtitle__jnooi"))
+					contentSerieTitle = textContent(".Meta_subtitle__jnooi");
+
+				if (contentTitle || contentSerieTitle) {
+					presenceData.details = `Смотрит ${
+						!privacy ? contentTitle : contentSerieTitle ? "сериал" : "фильм"
+					}`;
+
+					contentTimestamps = presence.getTimestampsfromMedia(
+						document.querySelector("video")
+					);
+
+					[presenceData.startTimestamp, presenceData.endTimestamp] =
+						presence.getTimestamps(contentTimestamps[0], contentTimestamps[1]);
+				} else {
+					presenceData.details = "Смотрит телеканал";
+					presenceData.startTimestamp = browsingTimestamp;
+				}
+
+				isPaused = document.querySelector(".styles_play__lWZwM");
+
+				presenceData.state = contentSerieTitle;
+				presenceData.smallImageKey = isPaused ? Assets.Pause : Assets.Play;
+				presenceData.smallImageText = isPaused ? strings.pause : strings.play;
+
+				if (isPaused || !time) {
 					delete presenceData.startTimestamp;
 					delete presenceData.endTimestamp;
-				} else {
-					[presenceData.startTimestamp, presenceData.endTimestamp] =
-						presence.getTimestamps(video.currentTime, video.duration);
 				}
+			} else {
+				contentTitle = null;
+				contentSerieTitle = null;
 			}
+
```

Comment:
```suggestion
```

Reply:
For screenshots, could you show the discord activity at least?

Reply:
> For screenshots, could you show the discord activity at least?

Added screenshots to the description